ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/LoggingTests.kt

```diff
@@ -441,27 +441,6 @@ class LoggingTest : ClientLoader() {
         }
     }
 
-    @Test
-    fun testCustomServer() = clientTests(listOf("iOS", "native:CIO")) {
```

Comment:
can we replace this with a test server?

Reply:
We have similar tests with our internal server test server